core/controllers/collection_editor.py

```diff
@@ -140,9 +140,9 @@ class CollectionRightsHandler(CollectionEditorHandler):
     @acl_decorators.can_edit_collection
     def get(self, collection_id):
         """Gets the editing rights for the given collection."""
```

Comment:
Could we have doc-strings for this function. It might be required?

Reply:
I am not sure. That's how the doc-strings are for other handlers as well. I modified it to add the Args, but I don't know what else to do.

Reply:
I guess thats fine. I just thought we should args and return values.